target/linux/ath79/dts/ar9344_mikrotik_routerboard-951ui-2hnd.dts

```diff
@@ -70,5 +70,8 @@
 };
 
 &wmac {
-	qca,led-pin = /bits/ 8 <11>;
+	led {
+		led-sources = <11>;
+		led-active-low;
+	}
```

Comment:
Trailing semicolon missing here:

    Error: ../dts/ar9344_mikrotik_routerboard-951ui-2hnd.dts:77.1-2 syntax error
    FATAL ERROR: Unable to parse input tree

```suggestion
	};
```

Reply:
Fixed.

Reply:
Hello...